release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -675,6 +676,31 @@ In the table below, features are marked with the following statuses:
 |GA
 |GA
 
+|Pod-level bonding for secondary networks
+|TP
+|GA
+|GA
+
```

Comment:
Using this first entry as an example.
Some feedback I received from the Accessibility open office hours. 

- We should spell out the abbreviations the first time they are mentioned in each table
- We should maybe spell out TP GA etc. because the main concern is the "empty" cells with a -. We should replace the - with Not Available

```suggestion
|Pod-level bonding for secondary networks
|Technology Preview
|General Availability
|General Availability
```